api/api-doc/storage_service.json

```diff
@@ -2574,6 +2574,70 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/tablets/rebuild",
```

Comment:
I find the name confusing, since you can rebuild an existing replica, while this API also appends a new replica. A more apt name would be "add_replica", which also rebuilds it by default. Opinions @bhalevy @avikivity ?

Reply:
`add_replica` is fine. And indeed it can be implemented in a number of ways.
`replicate` could also be used for that IMO

Reply:
> I find the name confusing, since you can rebuild an existing replica, while this API also appends a new replica.

From previous discussions I judged that "rebuild an existing replica" is called "repair", while "rebuild" is about "(re)creating one more replica out of other nodes' data". But I don't insist, add_replica or replicate (as Benny suggest) is good name too